Index: krusader-2.0.0/krusader/VFS/kiojobwrapper.cpp
===================================================================
--- krusader-2.0.0/krusader/VFS/kiojobwrapper.cpp
+++ krusader-2.0.0/krusader/VFS/kiojobwrapper.cpp	2009-07-26 03:47:30.829844134 +0300
@@ -41,6 +41,7 @@
 #include <klocale.h>
 #include "virtualcopyjob.h"
 #include <QTextDocument>
+#include <cstdio>
 
 class JobStartEvent : public QEvent {
 public:
