--- krusader/Panel/krvfsmodel.cpp.gcc	2010-12-07 08:52:34.000000000 +0000
+++ krusader/Panel/krvfsmodel.cpp	2010-12-07 08:53:12.000000000 +0000
@@ -609,7 +609,7 @@
 				return compareTexts(file1->vfile_getName(), file2->vfile_getName(), sp->properties(), sp->isAscending(), true);
 			return compareTexts( sp->data(), sp2->data(), sp->properties(), sp->isAscending(), true );
 	}
-	file1->vfile_getName() < file2->vfile_getName();
+	return file1->vfile_getName() < file2->vfile_getName();
 }
 
 bool itemGreaterThan( SortProps *sp, SortProps *sp2 )
