diff -Nurpa -x '*~' -x '*.orig' -x '*.rej' -x '*.swp' krusader-2.0.0/krusader/VFS/normal_vfs.cpp newdir/krusader/VFS/normal_vfs.cpp
--- krusader-2.0.0/krusader/VFS/normal_vfs.cpp	2009-04-11 22:40:15.000000000 +0300
+++ krusader-2.0.0/krusader/VFS/normal_vfs.cpp	2010-04-21 01:36:30.368399934 +0300
@@ -251,6 +251,9 @@ vfile* normal_vfs::vfileFromName(const Q
 	KDE_struct_stat stat_p;
 	stat_p.st_size = 0;
 	stat_p.st_mode = 0;
+	stat_p.st_mtime = 0;
+	stat_p.st_uid = 0;
+	stat_p.st_gid = 0;
 	KDE_lstat(fileName.data(),&stat_p);
 	KIO::filesize_t size = stat_p.st_size;
 	QString perm = KRpermHandler::mode2QString(stat_p.st_mode);
